llm/server.go

```diff
@@ -225,7 +225,7 @@ func NewLlamaServer(gpus discover.GpuInfoList, model string, ggml *GGML, adapter
 		fa = false
 	}
 
-	kvct := envconfig.KvCacheType()
+	kvct := strings.ToLower(envconfig.KvCacheType())
```

Comment:
Great! Should we do the same in `memory.go` ?

Reply:
Completely forgot about that. Done.